README.md

```diff
@@ -402,75 +464,71 @@ To give context to this CSS, here is an HTML snippet:
 
 Internally, this option is used when a `popstate` event triggers Pjax (to not `pushState()` again).
 
-##### `analytics` (Function | Boolean, default: a function that pushes `_gaq` `_trackPageview` or sends `ga` `pageview`
+### `analytics` (Function | Boolean, default: a function that pushes `_gaq` `_trackPageview` or sends `ga` `pageview`
 
 Function that allows you to add behavior for analytics. By default it tries to track
-a pageview with Google Analytics (if it exists on the page).
-It's called every time a page is switched, even for history navigation.
+a pageview with Google Analytics (if it exists on the page). It's called every time a page is switched, even for history navigation.
 
 Set to `false` to disable this behavior.
 
-##### `scrollTo` (Integer | \[Integer, Integer\] | False, default: `0`)
+### `scrollTo` (Integer | \[Integer, Integer\] | False, default: `0`)
 
 When set to an integer, this is the value (in px from the top of the page) to scroll to when a page is switched.
 
 When set to an array of 2 integers (\[x, y\]), this is the value to scroll both horizontally and vertically.
 
 Set this to `false` to disable scrolling, which will mean the page will stay in that same position it was before loading the new elements.
 
-##### `scrollRestoration` (Boolean, default: `true`)
+### `scrollRestoration` (Boolean, default: `true`)
 
-When set to true, attempt to restore the scroll position when navigating backwards or forwards.
+When set to `true`, Pjax attempt to restore the scroll position when navigating backwards or forwards.
 
-##### `cacheBust` (Boolean, default: `true`)
+### `cacheBust` (Boolean, default: `true`)
 
-When set to true, append a timestamp query string segment to the requested URLs
-in order to skip browser cache.
+When set to `true`, Pjax appends a timestamp query string segment to the requested URL in order to skip browser cache.
 
-##### `debug` (Boolean, default: `false`)
+### `debug` (Boolean, default: `false`)
 
 Enables verbose mode. Useful to debug page layout differences.
 
-##### `currentUrlFullReload` (Boolean, default: `false`)
+### `currentUrlFullReload` (Boolean, default: `false`)
 
-When set to true, clicking on a link that points to the current URL will trigger a full page reload.
+When set to `true`, clicking on a link that points to the current URL will trigger a full page reload.
 
-When set to `false`, clicking on such a link will cause Pjax to load the 
-current page like any page.
-If you want to add some custom behavior, add a click listener to the link,
-and set `preventDefault` to true, to prevent Pjax from receiving the event.
+When set to `false`, clicking on such a link will cause Pjax to load the
+current page like any page. If you want to add some custom behavior, add a click listener to the link and call `preventDefault()`. This will prevent Pjax from receiving the event.
```

Comment:
_'...like any other page...'_

Reply:
@robinnorth instead of `...like any other page...` here I reversed the language of the `true` example: `...without a full page reload...`.

Does that sound okay?